### ISSUE TITLE
Retrained models doesn't work

### ISSUE THREAD

**Issue by wilsonhws** [ccd97/image-classify-server#3]
Hi, I must say thank you for sharing the project but i've encountered some issues.
The original graph.pb and labels.txt works, but not my models(graph+labels).
Do you have a specific way to train your models ?

This is how I train my models:
https://codelabs.developers.google.com/codelabs/tensorflow-for-poets/#3

**Comment by ccd97** [comment]
As for the inception model I used, it had first layer named as `DecodeJpeg/contents:0` and softmax layer as `softmax:0`.
Changes the layer names if they are different.

**Comment by wilsonhws** [comment]
Thanks for the tips, it works perfectly.
